client/components/activity-card-list/visible-days-limit-upsell/index.tsx

```diff
@@ -74,27 +73,23 @@ const VisibleDaysLimitUpsell: React.FC< OwnProps > = ( { cardClassName } ) => {
 			<div className="visible-days-limit-upsell__next-activity">{ card }</div>
 			<div className="visible-days-limit-upsell__call-to-action">
 				<h3 className="visible-days-limit-upsell__call-to-action-header">
-					{ preventWidows(
-						translate(
-							'Restore backups older than %(retentionDays)d day',
-							'Restore backups older than %(retentionDays)d days',
-							{
-								count: visibleDays,
-								args: { retentionDays: visibleDays },
-							}
-						)
+					{ translate(
+						'Restore backups older than %(retentionDays)d day',
+						'Restore backups older than %(retentionDays)d days',
+						{
+							count: visibleDays,
+							args: { retentionDays: visibleDays },
+						}
 					) }
 				</h3>
 				<p className="visible-days-limit-upsell__call-to-action-copy">
-					{ preventWidows(
-						translate(
-							'Your activity log spans more than %(retentionDays)d day. Upgrade your backup storage to access activity older than %(retentionDays)d day.',
-							'Your activity log spans more than %(retentionDays)d days. Upgrade your backup storage to access activity older than %(retentionDays)d days.',
-							{
-								count: visibleDays,
-								args: { retentionDays: visibleDays },
-							}
-						)
```

Comment:
Not sure how to test this one, but should be safe given the text is wrapped in `p` and `h3` tags.

Reply:
For these sorts of ones, to test locally I wind up just commenting out the conditionals so that it'll appear to test it out quickly.

I.e. commenting out these lines: https://github.com/Automattic/wp-calypso/blob/d6fa1fef31db5bc621244b171a512feb7909c186/client/components/activity-card-list/visible-days-limit-upsell/index.tsx#L56-L58

And then setting this to always render:

https://github.com/Automattic/wp-calypso/blob/72748382b323948941e4e11811df1176fe6f26ee/client/my-sites/activity/activity-log/index.jsx#L552

This one's looking good to me!

<img width="966" alt="image" src="https://github.com/user-attachments/assets/c8b1a90b-7c47-4f3f-8c5c-66220daa29e6" />